conf/awsecr.config

```diff
@@ -0,0 +1,26 @@
+/*
+Config to set public AWS ECR images wherever possible
+This improves speed when running on AWS infrastructure.
+Use this as an example template when using your own private registry.
+*/
+
+docker.registry = 'public.ecr.aws'
+podman.registry = 'public.ecr.aws'
+
+process {
+    withName: '.*:SAMPLESHEET_CHECK' {
+        container = "public.ecr.aws/docker/library/python:3.8.16"
```

Comment:
This is a different container to the one used by the pipeline? We should use the same one but from `quay.io` if it doesn't exist on AWS ECR.
https://github.com/nf-core/smrnaseq/blob/81c1c0e22be35d09d0b1c660b5fb2ef7b7f40cea/modules/local/samplesheet_check.nf#LL9C10-L9C36

Reply:
Ideally, be nice to use Biocontainers wherever possible because this ensures we are using the same images for Singularity and Docker.